Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1222,6 +1222,20 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+Neighbor Discovery
+******************
+
+When kube-proxy replacement is enabled Cilium does L2 neighbor discovery of nodes in the cluster, in
+some setups Cilium may leave stale entries behind in the neighbor table causing ethernet frames
+between some nodes to drop. To prevent Cilium from performing this neighbor discovery and instead
```

Comment:
s/this neighbor discovery/the neighbor discovery/

Reply:
Addressed in https://github.com/cilium/cilium/commit/262176f973869ede633241f226d038cf60a81de4